src/macros.jl

```diff
@@ -65,24 +53,33 @@ function buildrefsets(expr::Expr, cname)
 end
 
 buildrefsets(c, cname)  = (cname, Any[], Any[], :())
+
+"""
+    JuMP.buildrefsets(expr::Expr)
+
+Helper function for macros to construct container objects. Takes an Expr that specifies the container, e.g. `:(x[i=1:3,[:red,:blue]],k=S; i+k <= 6)`, and returns:
+
+    1. `refcall`: Expr to reference a particular element in the container, e.g. :(x[i,red,s])
+    2. `idxvars`: Names for the index variables, e.g. [:i, gensym(), :k]
+    3. `idxsets`: Sets used for indexing, e.g. [1:3, [:red,:blue], S]
+    4. `condition`: Expr containing any conditional imposed on indexing, or :() if none is present
```

Comment:
Shouldn't we also use backsticks for the exemples. So "e.g. `:(x[i,red,s])`" instead of "e.g. :(x[i,red,s])"

Reply:
Yep